src/common/components/layouts/pages/status/JobStatus.js

```diff
@@ -67,10 +69,19 @@ function JobStatus({
     errorMessage,
     complete,
     onCancel,
+    onFileDrop,
     cancellingJob,
 }) {
     const { id: jobId } = useParams();
 
+    const onDrop = useCallback(
+        acceptedFiles => {
+            onCancel();
```

Comment:
Add `await` before `onCancel();`. Otherwise the cancel call fails due to the redirect to another page